avm/src/main.rs

```diff
@@ -48,7 +56,15 @@ fn parse_version(version: &str) -> Result<Version, Error> {
 }
 pub fn entry(opts: Cli) -> Result<()> {
     match opts.command {
-        Commands::Use { version } => avm::use_version(&version),
+        Commands::Use { version } => {
+            match version {
+                Some(v) => use_version(&v),
+                None => {
+                    let version_from_vavm = read_anchorversion_file()?;
+                    use_version(&version_from_vavm)
+                }
+            }
+        },
```

Comment:
Let's keep the structure the same and handle the logic inside `avm::use_version`.
```suggestion
        Commands::Use { version } => avm::use_version(&version),
```